### ISSUE TITLE
[BUG] Verbosity = 1, isProvideTrainingMetric= True, and model.saveNativeModel seemingly do nothing

### ISSUE THREAD

**Issue by sohaibsyed21** [microsoft/SynapseML#2312]
### SynapseML version

1.0.4

### System information

- **Language version** (e.g. python 3.8, scala 2.12): Python 3.9
- **Spark Version** (e.g. 3.2.3): 3.5.1
- **Spark Platform** (e.g. Synapse, Databricks): Amazon EMR

### Describe the problem

Working on a feature selection function using LightGBMRegressor() and according to documentation there are parameters named 'verbosity' and 'isProvideTrainingMetric' that I would assume print out information for training and evaluation metrics as the model trains. However, this is not the case. There are no errors raised, simply missing output or logs that I assume should be there. An additional detail is that I am not setting up an iteration loop; this is a single pass fit.

This leads to the next problem: I wanted to investigate the model by saving it to local disk with the .saveNativeModel() to ensure that even though I can not see the the output, I can save and load the model to local disk where I can do some quick testing, however, after training the model and using .saveNativeModel('path/to/file') , nothing happens.

### Code to reproduce issue

```python
    # Add a 'validation' column
    train_df_withVal = train_df.withColumn(
        'validation', 
        F.when(rand() < val_frac, 1).otherwise(0)
    )

    pandas_df_withVal = train_df_withVal.toPandas()
    pandas_df_withVal.to_excel(f"{model_output_archive_directory}/traindf_withVal.xlsx", index=False)


    model = LightGBMRegressor(**params, featuresCol='features',\
        labelCol=target_feature_name, seed=random_seed, featuresShapCol='shap_values',validationIndicatorCol='validation',verbosity=1,isProvideTrainingMetric= True).fit(train_df_withVal)

    model.saveNativeModel('/mnt/model')
```

### What component(s) does this bug affect?

- [ ] `area/cognitive`: Cognitive project
- [ ] `area/core`: Core project
- [ ] `area/deep-learning`: DeepLearning project
- [x] `area/lightgbm`: Lightgbm project
- [ ] `area/opencv`: Opencv project
- [ ] `area/vw`: VW project
- [ ] `area/website`: Website
- [ ] `area/build`: Project build system
- [ ] `area/notebooks`: Samples under notebooks folder
- [ ] `area/docker`: Docker usage
- [x] `area/models`: models related issue

### What language(s) does this bug affect?

- [ ] `language/scala`: Scala source code
- [x] `language/python`: Pyspark APIs
- [ ] `language/r`: R APIs
- [ ] `language/csharp`: .NET APIs
- [ ] `language/new`: Proposals for new client languages

### What integration(s) does this bug affect?

- [ ] `integrations/synapse`: Azure Synapse integrations
- [ ] `integrations/azureml`: Azure ML integrations
- [ ] `integrations/databricks`: Databricks integrations

**Comment by stupidoge** [comment]
I think so many people asked the same question, how to see the loss curve. I believe the verbosity does not relate to this. (although I also want that function

**Comment by JoseRiosCervepar** [comment]
Leaving a comment just in case there's an update about this.

**Comment by stupidoge** [comment]
@mhamilton723 Hi Mark, anyone could help us solve this issue? I think this issue has existed for a so long time, but still cannot see any solution here.

**Comment by ranadeepsingh** [comment]
Thank you for reporting this issue. This is a duplicate of issue #2269, which tracks the problem where setting isProvideTrainingMetric=True does not surface per-iteration training metrics or loss curves to the PySpark driver. Please follow issue #2269 for updates on this feature gap. We are closing this thread as a duplicate.
